syftlib/src/test/resources/proto_files/plan.json

```diff
@@ -0,0 +1,1554 @@
+{
+  "id": {
+    "idInt": "85993941842"
+  },
+  "operations": [
+    {
+      "command": "t",
+      "ownerPlaceholder": {
+        "id": {
+          "idInt": "63605762519"
+        },
+        "tags": [
+          "#input-4",
+          "#1"
+        ]
+      },
+      "returnPlaceholders": [
```

Comment:
Do we need to worry about individual actions? I thought we would be getting a torchscript directly

Reply:
Yes, this should go. I was using it when trying to understand what was going on.